### ISSUE TITLE
Man pages has not been released 

### ISSUE THREAD

**Issue by 0xTadash1** [NerdyPepper/dijo#116]
There is a man pages file, but it does not seem to be included in each version release.
I think it will make people aware of the existence of the man pages, and also make itself more easier to get.

**Comment by NerdyPepper** [comment]
Yeah you are right, I'd like to ship man pages with the release as well, unfortunately they are outdated and I don't really know how to update them right now (I should familiarize myself with the format). 

It would be great if someone could update and help maintain them!

**Comment by salastro** [comment]
I will work on updating them some time later. Additionally, you should really [learn about the format](https://www.youtube.com/watch?v=r2JrbyezhM4); it is quite easy.
